src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/LabelExtensions.cs

```diff
@@ -4,7 +4,28 @@ namespace Xamarin.Platform
 {
 	public static class LabelExtensions
 	{
-		public static void UpdateText(this TextView textView, ILabel label) =>
+		public static void UpdateText(this TextView textView, ILabel label)
+		{
 			textView.Text = label.Text;
+		}
+
+		public static void UpdateTextColor(this TextView textView, ILabel label)
+		{
+			textView.UpdateTextColor(label, null);
+		}
+
+		public static void UpdateTextColor(this TextView textView, ILabel label, Forms.Color? lastUpdateColor)
```

Comment:
should this be called defaultColor?

Reply:
Can we remove and be like ios and just use the default inside?

Reply:
Oh, yeah, it should. That's left over from something else that was here.